src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -121,11 +121,7 @@ private AssertionMessages() {
   protected WebElement veToolMenu;
   @FindBy(css = "h3[id='headerWikis']")
   protected WebElement headerWhereIsMyExtensionPage;
-  @FindAll({
-          @FindBy(css = "#globalNavigation"),
-          @FindBy(css = ".site-head.no-shadow"),
```

Comment:
Avoid using cross platform selectors with @FindAll because Selenium is waiting 30s (?) to find them on a page